experiments/word2vec_evolution/store_in_arango.py

```
@@ -0,0 +1,217 @@
+        'num_chunks': embeddings['num_chunks'],
+        'processing_date': datetime.now().isoformat(),
+        'status': 'PROCESSED',
+        'embedding_dim': len(embeddings['paper_embedding']),
+        'paper_embedding': embeddings['paper_embedding']  # Store paper-level embedding
+    }
+    
+    # Store paper
+    papers_coll = db.collection('arxiv_papers')
+    try:
+        papers_coll.insert(paper_doc)
+        logger.info(f"  ✓ Paper document stored")
```

Ruff (0.12.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)